static/script.js

```diff
@@ -1,11 +1,32 @@
 $(function () {
+  window.setInterval(refresh, 30000)
+
   $('[data-localtime="true"]').each(function () {
     let $element = $(this)
     let dateTime = moment.utc($element.text().trim())
     dateTime.local()
     $element.text(dateTime.format('YYYY-MM-DD HH:mm:ss'))
   })
+
   $('.no-collapsable').on('click', function (event) {
     event.stopImmediatePropagation()
   })
+  $('#auto-refresh').on('change', function (event) {
+    if (localStorage.getItem('refresh_f')) {
```

Comment:
This IF-statement should verify If check box state is on/off. What is in local storage is irrelevant.
Try to very fast click on and off and you will see that this script desynchronize with button state.